### ISSUE TITLE
How to show event data in other controls?

### ISSUE THREAD

**Issue by premy** [jestrux/jquery-mini-event-calendar#23]
Not really an issue, but  I'm trying to show the event data in some other controls on the page, like this:

```javascript
function showEvent(event) {
    var eventTitle2 = document.getElementById("#title"); //select other element
    if(event && event !== null && event !== undefined){
        eventTitle.text(event.title);
        eventTitle2.text(event.title); //set title for other element
        eventsLink.text("");
        eventsLink.attr("href", event.link);
    }else{
        eventTitle.text("Sem Aulas");
        eventsLink.text("");
        eventsLink.attr("href", settings.calendar_link);
    }
}
```
Basically I want to show the title in another h3 on the page also:

![titleNull2](https://user-images.githubusercontent.com/1387407/57036081-accbb100-6c29-11e9-99cd-b2e308a7a0ed.jpg)

But I get a null on var eventTitle2:

![titleNull](https://user-images.githubusercontent.com/1387407/57035060-83aa2100-6c27-11e9-877f-5532df606940.jpg)

Please give me a hint on what I'm doing wrong here?

**Comment by jestrux** [comment]
@premy 
First of there's a typo in your selector: 
It should be `document.getElementById("title")`....without the **#** symbol.

Secondly when you select an element using document.getElementById, you get a dom element, and you're trying to set it's text using a jQuery method **.text()**, that won't work.

To fix it you can either:
 1. change your selector to be a jQuery selector like: **$("#title")**   or
 1. Set the text using vanilla javascript: **eventTitle2.innerText = event.title**

Since you're already using jQuery everywhere else, I would suggest using the first solution

**Comment by premy** [comment]
Thanks a lot mate, and sorry for bothering you with such silly problems, for these are things I actually "know"; just seem to have forgotten as I haven't done front end dev in quite a while. 
But anyway, almost finished the web app  I'm working on. If you'd like to see how I implemented your calendar there: http://exclusivoenem.com.br
Style still needs some polishing, maybe, but overall I think it looks and works good. On desktop anyway; need some additional tweaks for mobile browsers.
I added a link to this repo in the code source.
